core/templates/pages/subtopic-viewer-page/subtopic-viewer-page.mainpage.html

```diff
@@ -1,32 +0,0 @@
-<!DOCTYPE html>
-<html ng-app="oppia" itemscope itemtype="http://schema.org/Organization">
-  <head>
-    @load('base-components/header.template.html', {"title": "Oppia"})
-  </head>
-  <body>
-    <oppia-root>
-      <div ng-controller="Base" dir="<[direction]>">
-        <oppia-base-content>
-          <navbar-pre-logo-action>
-            <subtopic-viewer-navbar-pre-logo-action>
-            </subtopic-viewer-navbar-pre-logo-action>
-          </navbar-pre-logo-action>
-
-          <navbar-breadcrumb>
-            <subtopic-viewer-navbar-breadcrumb>
```

Comment:
Do we need to include a navbar?

Reply:
@Nik-09  I'm sorry, I don't necesaarily understand this comment, can you please elaborate? Thanks!

Reply:
Hi, @Lawful2002 the old html page also contains a navbar right i.e. "subtopic-viewer-navbar-breadcrumb". So do we also need to include the navbar in the file `core/templates/pages/subtopic-viewer-page/subtopic-viewer-page-root.component.html`?

Does this help?
Thank you

Reply:
Done! (Apologies for the late reply)

Reply:
Thank you @Lawful2002